OPAL/ide/src/main/scala/org/opalj/ide/problem/EdgeFunction.scala

```diff
@@ -57,7 +57,11 @@ case class IdentityEdgeFunction[Value <: IDEValue]() extends EdgeFunction[Value]
     }
 
     override def equalTo(otherEdgeFunction: EdgeFunction[Value]): Boolean =
-        (otherEdgeFunction eq this) || otherEdgeFunction.isInstanceOf[IdentityEdgeFunction[Value]]
+        (otherEdgeFunction eq this) ||
+            (otherEdgeFunction match {
+                case IdentityEdgeFunction() => true
```

Comment:
Better: `case _: IdentityEdgeFuction`

But given this case class has no fields, shouldn't it be a `case object` anyway so the equality boils down to `otherEdgeFunction eq this`?